### PR TITLE
Tim/update activitity interaction documentation

### PR DESCRIPTION
**Pull request by timaccelo**
This code updates documentation about interactions on activities to better represent the current functionality of the public API.

### CODE REVIEW

File: source/includes/endpoints/_activities.md [hunk 1]
```diff
@@ -234,13 +234,13 @@ Activity threads are described by the thread object, this contains the following
...
 
-In the context of activities, an interaction is a recipient or sender of an activity, this will either be a staff, or
-affiliation or contact object. An activity can have several interactions; an email may be sent to several staff members
-and an affiliation. The form of interactions, and the way they are handled vary depending on the context, thus we deal
+In the context of activities, an interaction is a recipient or sender of an activity, this will either be a staff 
+affiliation or contact object. An activity can have several interactions; an email may be sent to several staff members 
+and/or affiliations. The form of interactions, and the way they are handled vary depending on the context, thus we deal
 with each case individually:
 
-1. [Interactions with a list of activities](#get-activities-handling-interacts)
```
Comment (timaccelo) [inline source/includes/endpoints/_activities.md]: These links were broken, I have linked to where I think they should have been going. Open to suggestions though as it is slightly ambiguous.

Comment (matthewhughes112) [reply source/includes/endpoints/_activities.md]: This looks fine, I'll just claim I knew what I was talking about and meant to link to that all along....

File: source/includes/endpoints/_activities.md [hunk 2]
```diff
@@ -952,7 +953,7 @@ for information on sending JSON requests) and including the following objects:
 | cc | Anyone to whom the activity should be cc'd.  |
 | bcc | Anyone to whom the activity should be bcc'd|
 
-Each object accepts a staff or affiliation (identified by their id) or a general email.
```
Comment (matthewhughes112) [inline source/includes/endpoints/_activities.md]: I think it should still be possible to pass a general email here?

File: source/includes/endpoints/_activities.md [hunk 3]
```diff
@@ -234,13 +234,13 @@ Activity threads are described by the thread object, this contains the following
...
-affiliation or contact object. An activity can have several interactions; an email may be sent to several staff members
-and an affiliation. The form of interactions, and the way they are handled vary depending on the context, thus we deal
+In the context of activities, an interaction is a recipient or sender of an activity, this will either be a staff 
+affiliation or contact object. An activity can have several interactions; an email may be sent to several staff members 
+and/or affiliations. The form of interactions, and the way they are handled vary depending on the context, thus we deal
 with each case individually:
 
-1. [Interactions with a list of activities](#get-activities-handling-interacts)   
-2. [List of interactions with a single activity](#get-activities-id-interacts)
+1. [Interactions with a list of activities](#handling-interacts)
```
[the remark this answers is not in the document]
Comment (timaccelo) [reply source/includes/endpoints/_activities.md]: Updated in ef59ead

File: source/includes/endpoints/_activities.md [hunk 4]
```diff
@@ -234,13 +234,13 @@ Activity threads are described by the thread object, this contains the following
 
 #### Activity Interactions
 
-In the context of activities, an interaction is a recipient or sender of an activity, this will either be a staff, or
-affiliation or contact object. An activity can have several interactions; an email may be sent to several staff members
-and an affiliation. The form of interactions, and the way they are handled vary depending on the context, thus we deal
+In the context of activities, an interaction is a recipient or sender of an activity, this will either be a staff
```
[the remark this answers is not in the document]
Comment (timaccelo) [reply source/includes/endpoints/_activities.md]: Updated in ef59ead

File: source/includes/endpoints/_activities.md (hunk 2 above)
Comment (timaccelo) [reply source/includes/endpoints/_activities.md]: Updated in ef59ead

File: source/includes/endpoints/_activities.md [hunk 5]
```diff
@@ -234,13 +234,13 @@ Activity threads are described by the thread object, this contains the following
 
 #### Activity Interactions
 
-In the context of activities, an interaction is a recipient or sender of an activity, this will either be a staff, or
-affiliation or contact object. An activity can have several interactions; an email may be sent to several staff members
-and an affiliation. The form of interactions, and the way they are handled vary depending on the context, thus we deal
+In the context of activities, an interaction is a recipient or sender of an activity, this will either be a staff,
```
Comment (matthewhughes112) [inline source/includes/endpoints/_activities.md]: Mind trimming the whitespace on this line too

File: source/includes/endpoints/_activities.md [hunk 6]
```diff
@@ -952,7 +952,8 @@ for information on sending JSON requests) and including the following objects:
 | cc | Anyone to whom the activity should be cc'd.  |
 | bcc | Anyone to whom the activity should be bcc'd|
 
-Each object accepts a staff or affiliation (identified by their id) or a general email.
+Each object accepts a staff or affiliation (identified by their id) or a general email. 
+(Note: When using a general email, not interact object will be created.)
```
Comment (matthewhughes112) [inline source/includes/endpoints/_activities.md]: Typo I think; 'not' should be 'no'

File: source/includes/endpoints/_activities.md [hunk 7]
```diff
@@ -952,7 +952,8 @@ for information on sending JSON requests) and including the following objects:
 | cc | Anyone to whom the activity should be cc'd.  |
 | bcc | Anyone to whom the activity should be bcc'd|
```
Comment (matthewhughes112) [inline source/includes/endpoints/_activities.md]: In the example above (around line 930), should the `emails` key be a string or an array of strings?

File: source/includes/endpoints/_activities.md (hunk 5 above)
Comment (timaccelo) [reply source/includes/endpoints/_activities.md]: Updated in 38f3a90

File: source/includes/endpoints/_activities.md (hunk 7 above)
Comment (timaccelo) [reply source/includes/endpoints/_activities.md]: Updated in 38f3a90

File: source/includes/endpoints/_activities.md (hunk 6 above)
Comment (timaccelo) [reply source/includes/endpoints/_activities.md]: Updated in 38f3a90

**Pull request merged**